src/components/DatePicker/DatePicker.scss

```diff
@@ -30,36 +31,41 @@
 }
 
 .react-calendar__navigation {
-  height: 2em;
-  font-size: 200%;
-  color: #00d8a4;
-  background-color: $assignments-darker-color;
+  height: 4vw;
+  font-size: 2vw;
+  font-family: $assignments-roboto-font;
+  background-color:$assignments-font-color;
   border-radius: 10px 10px 0px 0px;
 
   button {
     min-width: 20px;
     background: none;
-    color: black;
+    color: white;
+    font-weight: bold;
   }
 }
 .react-calendar__navigation .react-calendar__navigation button:enabled:hover,
 .react-calendar__navigation button:enabled:focus {
-  background-color: #e6e6e6;
+  background-color:$assignments-font-color;
 }
 .react-calendar__navigation button[disabled] {
-  background-color: #e6e6e6;
+  color: white;
+  background-color:$assignments-font-color;
 }
 .react-calendar__month-view__weekdays {
   text-align: center;
   text-transform: uppercase;
   color: #00d8a4;
   background-color: $assignments-darker-color;
   font-weight: bold;
-  font-size: 1em;
+  font-size: 1vw;
   color: black;
 }
 .react-calendar__month-view__weekdays__weekday {
   padding: 0.5em;
+  background-color:$assignments-font-color;
+  color: white;
+  font-size: 1rem;
```

Comment:
what an interesting font size choice bc doesnt this just make it the same size as the parent

Reply:
same size as the root parent i think not the direct?

Reply:
if its not there, then the weekdays on an ipad are like hella tiny